Makefile

```diff
@@ -13,13 +13,10 @@ VERSION ?= $(GIT_VERSION)
 IMAGE ?= $(REPO):$(VERSION)
 BASE_IMAGE ?= public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nonroot:latest.2
 GOLANG_VERSION ?= $(shell cat .go-version)
-BUILD_IMAGE ?= public.ecr.aws/bitnami/golang:$(GOLANG_VERSION)
+BUILD_IMAGE ?= public.ecr.aws/docker/library/golang:$(GOLANG_VERSION)
 GOARCH ?= amd64
 PLATFORM ?= linux/amd64
 
-export GOSUMDB = sum.golang.org
```

Comment:
We don't need to set these variables as we are building the binaries using the latest golang patch version for the family..  These variables are already set in the go env variables inside the container

```
GOSUMDB='sum.golang.org'
GOTOOLCHAIN='local'
GOVERSION='go1.22.2'
``` 
More context here - https://go.dev/doc/toolchain
```
The go command selects the Go toolchain to use based on the GOTOOLCHAIN setting. To find the GOTOOLCHAIN setting, the go command uses the standard rules for any Go environment setting:

If GOTOOLCHAIN is set to a non-empty value in the process environment (as queried by [os.Getenv](https://go.dev/pkg/os/#Getenv)), the go command uses that value.

Otherwise, if GOTOOLCHAIN is set in the user’s environment default file (managed with [go env -w and go env -u](https://go.dev/cmd/go/#hdr-Print_Go_environment_information)), the go command uses that value.

Otherwise, if GOTOOLCHAIN is set in the bundled Go toolchain’s environment default file ($GOROOT/go.env), the go command uses that value.
```

Go toolchain local will fall back to using what's packed in the local go environment.. i.e. the patch version of the image that was pulled